src/pages/signup.astro

```diff
@@ -0,0 +1,24 @@
+<html lang='en'>
+	<head>
+		<meta charset='utf-8' />
+		<link rel='icon' type='image/svg+xml' href='/favicon.svg' />
+		<meta name='viewport' content='width=device-width' />
+		<meta name='generator' content={Astro.generator} />
+		<title>Welcome to Go 👍 or 👎 No</title>
+	</head>
+
+	<body>
```

Comment:
I believe we can decrease some of the duplication between this and the other pages by using [astro's layouts](https://docs.astro.build/en/core-concepts/layouts/) . The goal would be to create a layout (AccountLayout?, WelcomeLayout?, not sure the best name) that would hold the html and header tags and we can inject whatever content the page needs.